app/queries/servers/custom_profile.ts

```diff
@@ -0,0 +1,302 @@
+        Q.where('id', fieldId),
+        Q.take(1),
+    ).observe().pipe(
+        switchMap((result) => (result.length ? result[0].observe() : of$(undefined))),
+    );
+};
+
+/**
+ * Observe a custom profile field
+ * @param database - The database instance
+ * @returns Observable of the custom profile field
+ */
+export const observeCustomProfileFields = (database: Database) => {
+    return database.get<CustomProfileFieldModel>(CUSTOM_PROFILE_FIELD).query(
+        Q.where('delete_at', 0),
+    ).observeWithColumns(['update_at', 'delete_at']).pipe(
+        distinctUntilChanged(),
+    );
+};
+
+/**
+ * Get a custom profile attribute by field ID and user ID
+ * @param database - The database instance
+ * @param fieldId - The field ID
+ * @param userId - The user ID
+ * @returns The custom profile attribute or undefined
+ */
+export const getCustomProfileAttribute = async (database: Database, fieldId: string, userId: string) => {
+    try {
+        const attributeRecord = await database.get<CustomProfileAttributeModel>(CUSTOM_PROFILE_ATTRIBUTE).query(
+            Q.where('field_id', fieldId),
+            Q.where('user_id', userId),
+        ).fetch();
+        return attributeRecord[0];
+    } catch {
+        return undefined;
+    }
+};
+
+/**
+ * Observe a custom profile attribute
+ * @param database - The database instance
+ * @param fieldId - The field ID
+ * @param userId - The user ID
+ * @returns Observable of the custom profile attribute
+ */
+export const observeCustomProfileAttribute = (database: Database, fieldId: string, userId: string) => {
+    return database.get<CustomProfileAttributeModel>(CUSTOM_PROFILE_ATTRIBUTE).query(
+        Q.where('field_id', fieldId),
+        Q.where('user_id', userId),
+        Q.take(1),
+    ).observe().pipe(
+        switchMap((result) => (result.length ? result[0].observe() : of$(undefined))),
+    );
+};
+
+/**
+ * Observe all custom profile attributes for a user
+ * @param database - The database instance
+ * @param userId - The user ID
+ * @returns Observable of the custom profile attributes
+ */
+export const observeCustomProfileAttributesByUserId = (database: Database, userId: string): Observable<CustomProfileAttributeModel[]> => {
+    return database.get<CustomProfileAttributeModel>(CUSTOM_PROFILE_ATTRIBUTE).query(
+        Q.where('user_id', userId),
+    ).observeWithColumns(['value']).pipe(distinctUntilChanged());
+};
+
+/**
+ * Observe all custom profile attributes for multiple users
+ * @param database - The database instance
+ * @param userIds - Array of user IDs
+ * @returns Observable of the custom profile attributes
+ */
+export const observeCustomProfileAttributesByUserIds = (database: Database, userIds: string[]) => {
+    return database.get<CustomProfileAttributeModel>(CUSTOM_PROFILE_ATTRIBUTE).query(
+        Q.where('user_id', Q.oneOf(userIds)),
+    ).observe().pipe(
+        distinctUntilChanged(),
```

Comment:
Why is this needed? This is only observing the list and not changes in any particular column